src/_pytest/recwarn.py

```diff
@@ -322,10 +322,9 @@ def found_str():
             for w in self:
                 if not self.matches(w):
                     warnings.warn_explicit(
-                        str(w.message),
-                        w.message.__class__,  # type: ignore[arg-type]
-                        w.filename,
-                        w.lineno,
-                        module=w.__module__,
```

Comment:
Let's keep the `module` in the PR as-is, the fix for that should go in a separate PR (it's a separate issue).